tests/queries/0_stateless/03403_text_index_function_search.sql

```diff
@@ -29,16 +29,19 @@ SELECT id FROM tab WHERE searchAny(1, 'a'); -- { serverError ILLEGAL_TYPE_OF_ARG
 SELECT id FROM tab WHERE searchAll(1, 'a'); -- { serverError ILLEGAL_TYPE_OF_ARGUMENT }
 -- 2nd arg must be const String
 SELECT id FROM tab WHERE searchAny(message, toFixedString('b', 1)); -- { serverError ILLEGAL_TYPE_OF_ARGUMENT }
-SELECT id FROM tab WHERE searchAny(message, materialize('b')); -- { serverError ILLEGAL_COLUMN }
+SELECT id FROM tab WHERE searchAny(message, materialize('b')); -- { serverError ILLEGAL_TYPE_OF_ARGUMENT }
```

Comment:
Comment in l. 30 is now incorrect.

Reply:
Could you add a test where the 2nd arg is a non-const array? (see my other comment)